packages/components/src/utils/table/test-utils/types.ts

```diff
@@ -0,0 +1,4 @@
+export interface TestData {
+  name: string;
+  value: string;
+}
```

Comment:
if you don't use the beforeeach stuff I think you don't need this type at all...